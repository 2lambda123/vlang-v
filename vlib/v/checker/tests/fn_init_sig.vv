fn init(i int) {
}
fn init() int {
    return 1
}
