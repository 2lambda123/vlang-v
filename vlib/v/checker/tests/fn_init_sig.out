vlib/v/checker/tests/fn_init_sig.vv:1:1: error: fn `init` cannot have parameters
    1 | fn init(i int) {
      | ~~~~~~~~~~~~~~
    2 | }
    3 | fn init() int {
vlib/v/checker/tests/fn_init_sig.vv:3:1: error: fn `init` cannot have a return type
    1 | fn init(i int) {
    2 | }
    3 | fn init() int {
      | ~~~~~~~~~~~~~
    4 |     return 1
    5 | }
